packages/teams-js/src/public/interfaces.ts

```diff
@@ -1262,3 +1262,41 @@ export interface ClipboardParams {
   /** Blob content in Base64 string format */
   content: string;
 }
+
+/**
+ * @hidden
+ * List of enter focus action items
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum EnterFocusActionItem {
+  /**
+   * Determines the direction to focus in app.
+   */
+  PreviousLandmark,
+  NextLandmark,
+  /**
+   * Determines if the focus should go to the particular content of the app.
+   */
+  Read,
+  Compose,
+}
+
+/**
+ * Return focus action items
+ */
+export enum ReturnFocusActionItem {
+  /**
+   * Determines the direction to focus in host for previous landmark.
+   */
+  PreviousLandmark,
```

Comment:
Is the name using 'landmark' based on parity with existing code? These appear to be focus stops or focus targets.

Reply:
It used to be true/false for navigateForward or navigateBackward , however with the new enum attribute we are differentiating between these two. Going forward once we have major version released, navigateFoward will be discontinued